VimeoNetworking/VimeoNetworking/AppConfiguration.swift

```diff
@@ -8,11 +8,17 @@
 
 import Foundation
 
-public let KeychainServiceVimeo = "Vimeo"
-public let KeychainServiceCameo = "Cameo"
-
+/**
+ *  Stores all static information relevant to a client application
+ */
 public struct AppConfiguration
 {
+        /// Keychain identifier for the Vimeo application
+    public static let KeychainServiceVimeo = "Vimeo"
```

Comment:
Isn't it technically the same application by bundle identifier? I think they can both just use Vimeo